src/poc/miner_poc_mgr.erl

```diff
@@ -544,8 +546,8 @@ initialize_poc(BlockHash, POCStartHeight, Keys, Vars, Ledger, #state{pub_key = C
                         start_height = POCStartHeight
                     },
                     ok = write_local_poc(LocalPOC, State),
-                    lager:info("started poc for challengeraddr ~p, target: ~p, onionhash ~p",
-                        [?TO_ANIMAL_NAME(Challenger), OnionKeyHash]),
+                    lager:info("started poc at blockheight ~p for challengeraddr ~p, onionhash ~p, target: ~p",
```

Comment:
fyi, I've started calling it poc_id instead of onion_key_hash

Reply:
on the miner side, poc_id is the unhashed version of the onion key

Reply:
then unhashed version is the poc_public_key :-)